Sources/Testing/Issues/Issue.swift

```diff
@@ -32,27 +32,11 @@ public struct Issue: Sendable {
     ///   - expected: The expected number of times
     ///     ``Confirmation/confirm(count:)`` should have been called.
     ///
-    /// This issue can occur when calling
-    /// ``confirmation(_:expectedCount:isolation:sourceLocation:_:)`` when the
-    /// confirmation passed to these functions' `body` closures is confirmed too
-    /// few or too many times.
-    indirect case confirmationMiscounted(actual: Int, expected: Int)
-
-    /// An issue due to a confirmation being confirmed the wrong number of
-    /// times.
-    ///
-    /// - Parameters:
-    ///   - actual: The number of times ``Confirmation/confirm(count:)`` was
-    ///     actually called.
-    ///   - expected: The expected number of times
-    ///     ``Confirmation/confirm(count:)`` should have been called.
-    ///
-    /// This issue can occur when calling
-    /// ``confirmation(_:expectedCount:isolation:sourceLocation:_:)-9rt6m`` when
-    /// the confirmation passed to these functions' `body` closures is confirmed
-    /// too few or too many times.
-    @_spi(Experimental)
-    indirect case confirmationOutOfRange(actual: Int, expected: any RangeExpression & Sendable)
+    /// This issue can occur when calling ``confirmation(_:expectedCount:isolation:sourceLocation:_:)-5mqz2``
+    /// or ``confirmation(_:expectedCount:isolation:sourceLocation:_:)-6bkl6``
+    /// when the confirmation passed to these functions' `body` closures is
+    /// confirmed too few or too many times.
+    indirect case confirmationMiscounted(actual: Int, expected: any RangeExpression & Sendable)
```

Comment:
I'm noticing the hoops we have to jump through elsewhere in the PR for partial availability (e.g. `if #available(_parameterizedProtocolsAPI, *)`) and wondering if things would be simpler if we took the `RangeExpression` the user provided and stored its lower & upper bounds as discrete values (i.e. two associated values) in this enum case?

The upper bound could be `Optional`, too, to reinforce the invariant that we require a lower bound but not an upper, so that's arguably a stronger way to model it than attempting to preserve the original range value.

Reply:
Should be fine because the `_parameterizedProtocolsAPI` bit is only necessary for some operations on these protocols, see https://github.com/swiftlang/swift-evolution/blob/main/proposals/0353-constrained-existential-types.md.